app/src/main/java/com/example/workoutapp/WorkoutActivity.kt

```diff
@@ -9,10 +9,7 @@ import androidx.compose.foundation.lazy.LazyColumn
 import androidx.compose.foundation.lazy.items
 import androidx.compose.material.*
 import androidx.compose.material.icons.Icons
-import androidx.compose.material.icons.filled.Delete
-import androidx.compose.material.icons.filled.Home
-import androidx.compose.material.icons.filled.Lock
-import androidx.compose.material.icons.filled.Person
+import androidx.compose.material.icons.filled.*
```

Comment:
I'd leave this as it were. Wildcard imports (*) are considered bad practice for a couple of reasons:

- Less documentation on what classes are being imported.
- If you have two wildcard imports, and they have a class with the same name, there will be a collision. Example: java.util.* and java.sql.* both have a class named Date. Different classes, same name. So, if you're using wildcards, which Date class should be used?
- It makes no difference in runtime performance, as the compiler will automatically optimize imports.